o1vm/src/interpreters/mips/column.rs

```diff
@@ -159,3 +171,9 @@ impl ColumnIndexer for Instruction {
         Column::DynamicSelector(usize::from(self) - N_MIPS_REL_COLS)
     }
 }
+
+impl Instruction {
+    pub fn to_selector_column_idx(self) -> usize {
+        usize::from(self) - N_MIPS_REL_COLS
```

Comment:
This is not correct. You get an underflow.

Reply:
This is correct. `usize::from(self)` is not the conversion from `EnumIter` into a `usize`, it is actually an implementation from Instruction to `usize` defined above. Sorry.